bootstrap/main.go

```diff
@@ -24,6 +26,11 @@ type options struct {
 	Token     string `long:"token" description:"Weave Cloud token" required:"true"`
 }
 
+func init() {
+	// https://sentry.io/weaveworks/launcher-bootstrap/
+	raven.SetDSN("https://44cf71b08710447888c993011b1302fc:8f57948cabd34bbe854b196635bff59f@sentry.io/288665")
+}
+
 func main() {
```

Comment:
Capturing panics would be useful:
https://github.com/weaveworks/launcher/blob/master/agent/main.go#L154